src/router/logged-in/screens/HomeScreen.tsx

```diff
@@ -198,14 +197,25 @@ const HomeScreen = ({ navigation }) => {
                 </Text>
               </Text>
             </UrlButton>
-
             <UrlButton
               bgColor={Colors.backgroundAlt}
               href={privacyUrls[i18n.language] || privacyUrls.en}
             >
               <Text center>{t('privacyPolicy')}</Text>
             </UrlButton>
 
+            <CtaButton
+              onPress={() => navigation.navigate({ routeName: 'LoggedOut' })}
```

Comment:
I don't like navigating to `LoggedOut` and trusting that it ends up in the right place and uses `isLoggedIn` to come back.

Most of the LanguageScreen is currently in a UI component. We can create a new screen in `src/router/logged-in/screens/ChangeLanguageScreen.tsx`, which uses the same UI component. Then we can move the `onContinue` navigation logic from the UI component to the screen, since that's the only thing that's different between these screens.

Then we'll also have smoother screen transitions; pushing and popping the `change language` screen on the stack and supporting the back button/gesture.

Reply:
Sounds good. I never know how much you want to "pollute" the app with multilanguage code, so I was trying to reuse as much as possible. The proposal makes sense. I'll try to re-do it that way.